hack/make-rules/test-cmd-util.sh

```diff
@@ -1448,7 +1448,7 @@ run_kubectl_get_tests() {
   kube::test::if_has_string "${output_message}" 'valid-pod' # pod details
   output_message=$(kubectl get pods/valid-pod -o name -w --request-timeout=1 "${kube_flags[@]}")
   kube::test::if_has_not_string "${output_message}" 'STATUS' # no headers
-  kube::test::if_has_string     "${output_message}" 'pods/valid-pod' # resource name
+  kube::test::if_has_string     "${output_message}" 'Pod./valid-pod' # resource name
```

Comment:
make the output avoid dropping out a trailing dot

Reply:
Thanks, done